examples/routing_service/routing_service_with_security/USER_ROUTING_SERVICE.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0"?>
+<dds xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+     
+  xsi:noNamespaceSchemaLocation="https://community.rti.com/schema/6.0.0/rti_routing_service.xsd">
+    <!--
+    ********************************************************************** 
+    RTI Routing Service with Security example                                            
+    ********************************************************************** 
+    This example bridges data from Domain 2 (secured domain) to 
+	Domain 1 (unsecured domain)
+    ********************************************************************** 
+    -->
+
+    <!-- 
+    This is the routing service configuration for this example
+    -->
+    <routing_service name="security_example">
+
+        <annotation>
+            <documentation>
+                Bridges data from Domain 2 (secured) to Domain 1 (unsecured)
+            </documentation>
+        </annotation>
+
+        <domain_route name="DomainRoute" enabled="true">
+		    <!-- Participant 1 is in Domain 1, an unsecured domain -->
+            <participant name="1">
+                <domain_id>1</domain_id>
+            </participant>
+			
+			<!-- Participant 2 is in Domain 2, a secured domain -->
+            <participant name="2">
+                <domain_id>2</domain_id>
+              <participant_qos>
+                <property>
+                  <value>
+                    <element>
```

Comment:
Is this the recommended way to enable Security Plugins on Routing Service? Can't we make participant_qos inherit from the BuiltinQosLib::Generic.Security profile?

Reply:
participant qos inherit from security profile.